description: use python3 as the interpreter for the genheaders script
author: Michael Gilbert <mgilbert@debian.org>

--- a/xml/genheaders.py
+++ b/xml/genheaders.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 #
 # Copyright (c) 2013-2017 The Khronos Group Inc.
 #
